helpers/cookies.ts

```diff
@@ -17,19 +17,24 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import { memoize } from 'lodash';
 
-const parseCookies = memoize(
-  (documentCookie: string): T.Dict<string> => {
-    const rawCookies = documentCookie.split('; ');
-    const cookies: T.Dict<string> = {};
-    rawCookies.forEach(candidate => {
-      const [key, value] = candidate.split('=');
-      cookies[key] = value;
-    });
-    return cookies;
+// Memoize only last parsed element
```

Comment:
why only the last?

Reply:
We don't need more than this. We don't use cookies much so it's not so often that they change, they will usually do when you logout/login. But when you logout/login then the page is reloaded and the memoization cache is lost.
So in most case it's enough to just remember the last one, and if the cookie change without a page reload then we will cache it and it probably won't change anymore, and for sure won't go back to it's previous value (due to the nature of what we have in cookies).